docs/flow/modin/experimental/xgboost.rst

```diff
@@ -0,0 +1,132 @@
+parameter `num_actors`.
+
+.. _train:
+.. autofunction:: modin.experimental.xgboost.train
+    :noindex:
+
+Internal execution flow on Ray backend
+''''''''''''''''''''''''''''''''''''''
+
+Internal functions :ref:`_train <internal_train>` and :ref:`_predict <internal_predict>` work similary.
+Approximate execution flow of internal implementation is the next:
+
+1. The data arrives to :ref:`_train <internal_train>`/:ref:`_predict <internal_predict>` function as the
+   :ref:`DMatrix <DMatrix>` object. Using iterator of :ref:`DMatrix <DMatrix>`, lists of ``ray.ObjectRef``
+   with row partitions of Modin DataFrame are exctracted. Example:
+
+   .. code-block:: python
+
+     # Extract lists of row partitions from dtrain (DMatrix object)
+     X_row_parts, y_row_parts = dtrain
+   ..
+
+2. On this step parameter ``num_actors`` is processed. Internal function :ref:`_get_num_actors <get_num_actors>`
+   determines value provided by user and checks this to hitting in the set of values
+   (int, "default_train", "default_predict").
+
+   * int - ``num_actors`` won't be changed. This value will be used.
+   * "default_train" - ``num_actors`` will be computed using condition that 1 actor should use maximum 2 CPUs.
+     This condition was chosen for using maximum parallel workers with multithreaded xgboost training (2 threads
+     per worker will be used in this case).
+   * "default_predict" - ``num_actors`` will be computed using condition that 1 actor should use maximum 8 CPUs.
+     This condition was chosen to combine  parallelization techniques: parallel actors and parallel threads.
+
+**Note**: ``num_actors`` parameter is advisable to provide to public functions :ref:`train <train>` and ``predict``
+method of :ref:`Booster<Booster>` class to reach best performance for any own case.
+
+3. ``ray.util.placement_group`` is created to reserve of all available Ray resources. After that
+   :ref:`ModinXGBoostActor <Actor>` objects are created using resources of previously created placement group.
+
+4. Data (`dtrain` for :ref:`_train <internal_train>`, `data` for :ref:`_predict <internal_predict>` is split
+   between actors evenly. Internal function :ref:`_split_data_across_actors <split_data_across_actors>` runs
+   assigning of row partitions to actors using internal function :ref:`_assign_row_partitions_to_actors <assign_parts>`.
+   This function creates dictionary in the form: `{actor_rank: ([part_i0, part_i3, ..], [0, 3, ..]), ..}` - for training,
+   `{actor_rank: [part_i0, part_i1, ..], ..}` - for prediction.
+
+**Note**: :ref:`_assign_row_partitions_to_actors <assign_parts>` considers IP addresses of row partitions of `dtrain` data to
+minimize excess data transfer.
+
+5. For each actor object methods ``set_train_data`` or ``set_predict_data`` of :ref:`ModinXGBoostActor <Actor>` class
+   object are called remotely to run loading row partitions belonging to actor according to dictionary
```

Comment:
pls re-phrase, I cannot understand what this means :)

Reply:
It was re-phrased, please, check again)